elastic/kibana
# [RAM] Rule type params OAS verification and POC

## Summary
Depends on: https://github.com/elastic/kibana/pull/156357
Relates to https://github.com/elastic/kibana/issues/180056

POC to verify that we are able to generate Open API documentation for rule type schemas. This PR demonstrates this with the following rule types:

- Custom threshold
- Metric threshold
- ES query

To test:

- Run `yarn start` 
- Once ES and kibana has started, navigate to `http://localhost:5601/<3 letter sequence>/api/oas?pathStartsWith=/internal/alerting` or `http://localhost:5601/<<3 letter sequence>/api/oas?pathStartsWith=/api/alerting`
- Take the output and copy it to https://editor.swagger.io/
- Open API should generate documentation based on the JSON
